src/components/Post/HotPost/index.tsx

```diff
@@ -61,6 +64,9 @@ function HotPost(HotPostList: HotPostProps) {
               className={styles.link__button}
               key={link.id}
               href={link.url}
+              onClick={() => {
+                logger.actionEventClick({ actionTitle: 'CAMPUS', title: 'short_cut', value: link.title.join('') });
```

Comment:
따로 구분자는 필요없을까요? 언더바 같은게 있으면 더 보기 편할 거 같아요!
```suggestion
                logger.actionEventClick({ actionTitle: 'CAMPUS', title: 'short_cut', value: link.title.join('_') });
```

Reply:
좋아요! 이전보다 깔끔합니다!!👍